src/otx/core/data/transform_libs/torchvision.py

```diff
@@ -8,19 +8,51 @@
 from inspect import isclass
 from typing import TYPE_CHECKING, Any
 
+import PIL.Image
 import torch
 import torchvision.transforms.v2 as tvt_v2
 from lightning.pytorch.cli import instantiate_class
 from omegaconf import DictConfig
 from torchvision import tv_tensors
+from torchvision._utils import sequence_to_str
 from torchvision.transforms.v2 import functional as F  # noqa: N812
 
+from otx.core.data.entity.base import Points
+
 if TYPE_CHECKING:
     from torchvision.transforms.v2 import Compose
 
     from otx.core.config.data import SubsetConfig
 
 
+def custom_query_size(flat_inputs: list[Any]) -> tuple[int, int]:  # noqa: D103
+    sizes = {
+        tuple(F.get_size(inpt))
+        for inpt in flat_inputs
+        if tvt_v2._utils.check_type(  # noqa: SLF001
+            inpt,
+            (
+                F.is_pure_tensor,
+                tv_tensors.Image,
+                PIL.Image.Image,
+                tv_tensors.Video,
+                tv_tensors.Mask,
+                tv_tensors.BoundingBoxes,
+                Points,
```

Comment:
Is this problem coming from adding `Points`?

Reply:
An error occurred due to no `Points` type in the type checklist.

Reply:
https://github.com/pytorch/vision/blob/main/torchvision/transforms/v2/_utils.py#L177-L198